crates/ext-processor/src/headers.rs

```diff
@@ -43,36 +50,48 @@ mod tests {
     #[test]
     fn test_serialize_decision_sfv() -> Result<(), Box<dyn std::error::Error>> {
         assert_eq!(
-            serialize_decision_sfv(Decision {
-                accept: 0.0,
-                restrict: 0.0,
-                unknown: 1.0
-            })?,
-            "accept=0.0, restrict=0.0, unknown=1.0"
+            serialize_decision_sfv(
+                Decision {
+                    accept: 0.0,
+                    restrict: 0.0,
+                    unknown: 1.0
+                },
+                Outcome::Suspected
+            )?,
+            "accept=0.0, restrict=0.0, unknown=1.0, score=0.5, outcome=\"suspected\""
         );
         assert_eq!(
-            serialize_decision_sfv(Decision {
-                accept: 0.0,
-                restrict: 1.0,
-                unknown: 0.0
-            })?,
-            "accept=0.0, restrict=1.0, unknown=0.0"
+            serialize_decision_sfv(
+                Decision {
+                    accept: 0.0,
+                    restrict: 1.0,
+                    unknown: 0.0
+                },
+                Outcome::Restricted
+            )?,
+            "accept=0.0, restrict=1.0, unknown=0.0, score=1.0, outcome=\"restricted\""
         );
         assert_eq!(
-            serialize_decision_sfv(Decision {
-                accept: 1.0,
-                restrict: 0.0,
-                unknown: 0.0
-            })?,
-            "accept=1.0, restrict=0.0, unknown=0.0"
+            serialize_decision_sfv(
+                Decision {
+                    accept: 1.0,
+                    restrict: 0.0,
+                    unknown: 0.0
+                },
+                Outcome::Accepted
+            )?,
+            "accept=1.0, restrict=0.0, unknown=0.0, score=0.0, outcome=\"accepted\""
         );
         assert_eq!(
-            serialize_decision_sfv(Decision {
-                accept: 0.333,
-                restrict: 0.333,
-                unknown: 0.333
-            })?,
-            "accept=0.333, restrict=0.333, unknown=0.333"
+            serialize_decision_sfv(
+                Decision {
+                    accept: 0.333,
+                    restrict: 0.333,
+                    unknown: 0.333
+                },
+                Outcome::Suspected
```

Comment:
Similarly, this and the other `Outcome::Suspected` are both in the default `Outcome::Accepted` range. The default `Outcome::Suspected` range starts at 0.6. It probably isn't that important, but might help avoid confusion for anyone reading the tests.